packages/manager/modules/pci/src/components/project/quota-region-header/header.html

```diff
@@ -1,12 +1,21 @@
-<oui-header-tabs class="mb-2">
-    <oui-header-tabs-item
-        text="{{:: 'pci_project_quota_header' | translate}}"
-        state="pci.projects.project.quota"
-        state-params="{ projectId: $ctrl.projectId }">
-    </oui-header-tabs-item>
-    <oui-header-tabs-item
-        text="{{:: 'pci_project_region_header' | translate}}"
-        state="pci.projects.project.regions"
-        state-params="{ projectId: $ctrl.projectId }">
-    </oui-header-tabs-item>
-</oui-header-tabs>
+<oui-page-header class="mb-3">
+    <oui-header-tabs >
+        <oui-header-tabs-item
+            text="{{:: 'pci_project_quota_header' | translate}}"
```

Comment:
```suggestion
            data-text="{{:: 'pci_project_quota_header' | translate}}"
```

Reply:
Given the amount of page-header to update, I didn't pay attention to those

Reply:
Alright 👍